pages/learn/deploy/deploy-with-balena-button.md

```diff
@@ -38,17 +38,49 @@ You can further customize the behavior of the **Deploy with {{ $names.company.lo
 
 Through the use of a `balena.yml` config file, you may also provide application [configuration](/learn/manage/configuration/) and [environment](/learn/manage/serv-vars/) variables. If provided, they are pre-populated in the advanced modal dialog when using the **Deploy with {{ $names.company.lower }}** button.
 
+The `balena.yml` is can be used if you are submitting your app to [balenaHub][balenaHub].
```

Comment:
The `balena.yml` file can also be used to provide additional metadata to be used if the app is submitted to [balenaHub][balenaHub].